### PR TITLE
add git-land -- land without fetching any origin branch (except develop)

### PR DESCRIPTION
**Pull request by shofwan**

### CODE REVIEW

File: all-about-that-bash.sh [hunk 1]
```diff
@@ -879,4 +879,13 @@ try-land() {
...
 
-trap 'echo -e "${clPURPLE}-- Started at $(date +"%H:%M:%S") --${cLIGHTGRAY}"' DEBUG
\ No newline at end of file
+#pre-requisite: checked out on develop branch
+git-land() {
+    git fetch --no-tags origin develop; git rebase; git push;
+}
+
+git-land-multi() {
+    git-land; git-land; git-land; git-land; git-land;
```
Comment (haruntvlk) [inline all-about-that-bash.sh]: why not use param n to put the git-land into for-loop?

Comment (shofwan) [reply all-about-that-bash.sh]: how to do that :sweat_smile:

Comment (shofwan) [reply all-about-that-bash.sh]: forgive this fool apprentice, shifu :bowing_man:

Comment (haruntvlk) [reply all-about-that-bash.sh]:
```
loop-for() {
	for (( c=1; c<=${1}; c++ ))
	do  
	   echo "this is $c"
	done
}
```
${1} is the first param.
when executing 
```
loop-for 2
```
where ${1} = 2, the result will be:
```
this is 1
this is 2
```

Comment (shofwan) [reply all-about-that-bash.sh]: if it is just `loop-for`, what it will be look like?

Comment (shofwan) [reply all-about-that-bash.sh]: TL;DR I want to make just a single command.
 If you put number n, it will do n times, if you don't, it will do 1 time.

Comment (haruntvlk) [reply all-about-that-bash.sh]: then add condition before the code
```
n=1
if [ -n "$1" ]; then
   n=${1}
fi
for (( c=1; c<=n; c++ ))
...
```

**Comment by shofwan** [comment]
Updated yaa @nametal

File: all-about-that-bash.sh [hunk 2]
```diff
@@ -879,4 +879,25 @@ try-land() {
     git fetch --no-tags origin develop && git rebase origin/develop && arc land --onto develop
 }
 
-trap 'echo -e "${clPURPLE}-- Started at $(date +"%H:%M:%S") --${cLIGHTGRAY}"' DEBUG
\ No newline at end of file
+#pre-requisite: checked out on develop branch
```
Comment (haruntvlk) [inline all-about-that-bash.sh:882]: you can check it by using `$(getCurrentGitBranch)` function, it's provided on the same file.

Comment (shofwan) [reply all-about-that-bash.sh:882]: done @haruntvlk

**Review by haruntvlk — APPROVED** [review]

**Pull request merged**